src/openrct2/peep/Guest.cpp

```diff
@@ -6803,37 +6816,37 @@ struct item_pref_t
 
 // clang-format off
 static item_pref_t item_order_preference[] = {
```

Comment:
`item_pref_t` should be modified to use an `uint64_t` for item and remove of the `-ShopItem::Photo2` for the extra flags. You'll need to do a bit of work to make this work.